### ISSUE TITLE
Screen Reader focus is landing outside of location edit box when user selects any location.

### ISSUE THREAD

**Issue by xiaoyangliu1123** [Microsoft/BingMapsV8CodeSamples#41]
**Test Environment:**
URL: https://samples.bingmapsportal.com/?search=suggestion&sample=business-suggestions
Screen Reader: NVDA

**Repro Steps:**
Step 1: Open the above URL 
Step 2: Press Tab key and navigate to location edit box and type some character.
Step 3: Press down arrow key and select any desired location and observe the issue.
![image](https://github.com/microsoft/BingMapsV8CodeSamples/assets/152237890/f0b7e005-4d46-4c30-b677-aaecdebfdd3c)

**Actual Result:**
After selecting the location from the list items screen reader focus is landing outside of the location edit box. Screen reader shows "Business Suggestions - Bing Maps Samples document"
 
**Expected Result:**
After selecting from the list items, screen reader focus should not land outside of the pane, and it should be retained same on the location edit box.

**Comment by rbrundritt** [comment]
This appears to be limited to when the sample is ran inside of iframe. Running the same sample outside of that site works as expected. On the sample site, if you click on "Open in new tab" and try the scenario, you will see it works as expected. The behavior in question is a bug in the browsers iframe. Note that you may be able to work around this bug if needed by manually setting the focus in the suggestion selected callback function.
